src/governance.cpp

```diff
@@ -954,23 +954,22 @@ void CGovernanceManager::RequestGovernanceObject(CNode* pfrom, const uint256& nH
         return;
     }
 
-    CBloomFilter filter(GOVERNANCE_FILTER_ELEMENTS, GOVERNANCE_FILTER_FP_RATE, GetRandInt(999999), BLOOM_UPDATE_ALL);
+    CBloomFilter filter;
+    filter.clear();
 
     if(fUseFilter) {
         CGovernanceObject* pObj = FindGovernanceObject(nHash);
 
         if(pObj) {
+            int nFilterElements = min(3*mnodeman.CountMasternodes(), GOVERNANCE_FILTER_MAX_ELEMENTS); 
```

Comment:
As long as we have a larger load on testnet than on mainnet I don't see a problem.  We want to hit upper bounds on load as much as possible not try to replicate exact conditions.

I don't see much value in making testnet more efficient.